### PR TITLE
ODC-7535,ODC-7438: web-terminal automation improvement

### PR DESCRIPTION
**Pull request by The-Anton**
**Description:**

- Automation improvement for web-terminal package

**Jira Id:**
- Story: 
      -  [ODC-7535](https://issues.redhat.com/browse/ODC-7535) [ODC-7438](https://issues.redhat.com/browse/ODC-7438)

**Checks for approving Epic scenarios Automation PR:**
- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**
- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
    export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.13-042801.devcluster.openshift.com/
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p webterminal
```

**Screen shots**:
Screenshot 2024-04-11 at 12 19 30 PM

### CODE REVIEW

File: frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/common/webTerminal.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ function installDevWsAndReconfigureIdlingTimeout(dedicatedNamespace: string) {
   try {
     cy.exec(`oc login -u ${username}  -p ${password} --insecure-skip-tls-verify`);
     cy.exec(`oc new-project  ${dedicatedNamespace}`);
-    cy.exec(`oc apply -f testData/yamls/web-terminal/web-terminal.yaml -n ${dedicatedNamespace}`);
+    cy.exec(`oc apply -f testData/YAMLs/web-terminal/web-terminal.yaml -n ${dedicatedNamespace}`);
```
[the remark this answers is not in the document]
Comment (The-Anton) [reply frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/common/webTerminal.ts:25]: 👀 Sure, I will merge both of these folders.

File: frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-Terminal-devuser.ts [hunk 2]
```diff
@@ -3,40 +3,45 @@ import {
...
     cy.exec(`oc delete namespace ${nameSpace}`, { failOnNonZeroExit: true });
   },
 );
 
 When('user selects {string} from Project drop down menu', (projectName: string) => {
-  initTerminalPage.clickOnProjectDropDawn();
+  initTerminalPage.clickOnProjectDropDown();
   initTerminalPage.selectProject(projectName);
-  initTerminalPage.clickStartButton();
+  // initTerminalPage.clickStartButton();
```
[the remark this answers is not in the document]
Comment (The-Anton) [reply frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-Terminal-devuser.ts:46]: Yes, I will remove the line.

File: frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-teminal-basic.ts [hunk 3]
```diff
@@ -12,7 +17,7 @@ Then('user will see the terminal window opened in new tab', () => {
 });
 
 And('user does nothing with displayed terminal window 1 minutes', () => {
-  const terminalIdlingTimeout: number = Number(Cypress.env('TERMINAL_IDLING_TIMEOUT')) || 60000;
+  const terminalIdlingTimeout: number = Number(Cypress.env('TERMINAL_IDLING_TIMEOUT')) || 65000;
```
[the remark this answers is not in the document]
Comment (The-Anton) [reply frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-teminal-basic.ts:20]: I was thinking about adding a little more time to provide a cushion, but yes, it makes sense; we can stick with 60s.

**Comment by jerolimov** [comment]
/retitle ODC-7535,ODC-7438: web-terminal automation improvement

**Comment by sanketpathak** [comment]
Tests run green after the web terminal is installed manually
Screenshot 2024-04-24 at 10 54 47 PM
/lgtm

**Comment by sanketpathak** [comment]
PR edits only e2e tests
/label docs-approved
/label px-approved
/label qe-approved

**Pull request merged**